specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/auth.tests/errored.class/response.rpt

```diff
@@ -0,0 +1,10 @@
+
+accept http://localhost:8000/snmp
+accepted
+connected
+
+read method "GET"
+read header "Upgrade" "websocket"
+read header "Authorization" "Basic YWRtaW46YWRtaW4="
+
+#no response
```

Comment:
Similar comment as above.  We do not want to add tests that are 'buggy'.  The tests should perform according to spec.